guides/common/modules/proc_deleting-an-organization.adoc

```diff
@@ -3,8 +3,12 @@
 
 You can delete an organization if the organization is not associated with any lifecycle environments or host groups.
 If there are any lifecycle environments or host groups associated with the organization you are about to delete, remove them by navigating to *Administer* > *Organizations* and clicking the relevant organization.
-Do not delete the default organization created during installation because the default organization is a placeholder for any unassociated hosts in the {Project} environment.
+
+[IMPORTANT]
+====
+Do not delete *Default Organization* created during installation because the default organization is a placeholder for any unassociated hosts in the {Project} environment.
 There must be at least one organization in the environment at any given time.
+====
```

Comment:
```suggestion
ifndef::orcharhino[]
[IMPORTANT]
====
Do not delete *Default Organization* created during installation because the default organization is a placeholder for any unassociated hosts in the {Project} environment.
There must be at least one organization in the environment at any given time.
====
endif::[]
```

Reply:
The part about at least one organization being present applies to orcharhino too, I believe. So I tweaked the ifndef to apply to line 16 only.